ubuntu-focal-standard: macaulay2 needs bison

```
 [macaulay2-1.24.11]   [spkg-install] CMake Error at /sage/local/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  [macaulay2-1.24.11]   [spkg-install]   Could NOT find BISON (missing: BISON_EXECUTABLE)
```
https://github.com/passagemath/passagemath/actions/runs/12229492505/job/34109543735?pr=527#step:11:3628

--- a/build/pkgs/_prereq/distros/alpine.txt
+++ b/build/pkgs/_prereq/distros/alpine.txt
@@ -18,3 +18,5 @@ gcc
 g++
 ca-certificates
 coreutils
+flex
+bison

--- a/build/pkgs/_prereq/distros/conda.txt
+++ b/build/pkgs/_prereq/distros/conda.txt
@@ -14,3 +14,5 @@ perl
 python
 tar
 bc
+flex
+bison

--- a/build/pkgs/_prereq/distros/debian.txt
+++ b/build/pkgs/_prereq/distros/debian.txt
@@ -13,6 +13,8 @@ m4
 perl
 # flex is needed to build gcc
 flex
+# bison is needed by macaulay2
+bison
 # python3-minimal is not enough on debian buster, ubuntu bionic - it does not have urllib
 # system python for bootstrapping the build
 python3

--- a/build/pkgs/_prereq/distros/fedora.txt
+++ b/build/pkgs/_prereq/distros/fedora.txt
@@ -31,3 +31,5 @@ diffutils
 perl-IPC-Cmd
 # Needed to build gcc
 flex
+# bison is needed by macaulay2
+bison

--- a/build/pkgs/_prereq/distros/freebsd.txt
+++ b/build/pkgs/_prereq/distros/freebsd.txt
@@ -16,3 +16,5 @@ automake
 bash
 dash
 python # python metaport for bootstrapping the build
+flex
+bison

--- a/build/pkgs/_prereq/distros/gentoo.txt
+++ b/build/pkgs/_prereq/distros/gentoo.txt
@@ -27,3 +27,5 @@ dev-libs/libxml2
 sys-apps/findutils
 sys-apps/which
 sys-apps/diffutils
+sys-devel/flex
+sys-devel/bison

--- a/build/pkgs/_prereq/distros/nix.txt
+++ b/build/pkgs/_prereq/distros/nix.txt
@@ -16,3 +16,5 @@ gnutar
 bc
 gcc
 bash
+flex
+bison

--- a/build/pkgs/_prereq/distros/opensuse.txt
+++ b/build/pkgs/_prereq/distros/opensuse.txt
@@ -26,3 +26,7 @@ gzip
 # Issue #32368
 findutils
 diffutils
+# flex is needed to build gcc
+flex
+# bison is needed by macaulay2
+bison

--- a/build/pkgs/_prereq/distros/slackware.txt
+++ b/build/pkgs/_prereq/distros/slackware.txt
@@ -28,3 +28,5 @@ flex
 ca-certificates
 libxml2
 cyrus-sasl
+# bison is needed by macaulay2
+bison

--- a/build/pkgs/_prereq/distros/void.txt
+++ b/build/pkgs/_prereq/distros/void.txt
@@ -32,3 +32,7 @@ libffi-devel
 zlib-devel
 # to build python3 with the crypto module; there may be other packages that need it
 libxcrypt-devel
+# flex is needed to build gcc
+flex
+# bison is needed by macaulay2
+bison

